server/src/apub/community_inbox.rs

```diff
@@ -42,28 +33,36 @@ pub async fn community_inbox(
 fn handle_follow(
   follow: &Follow,
   request: &HttpRequest,
-  conn: &PgConnection,
+  community_name: &str,
+  db: DbPoolParam,
+  _chat_server: ChatServerParam,
 ) -> Result<HttpResponse, Error> {
   let user_uri = follow
     .follow_props
     .get_actor_xsd_any_uri()
     .unwrap()
     .to_string();
-  let user = fetch_remote_user(&Url::parse(&user_uri)?, conn)?;
-  verify(&request, &user.public_key.unwrap())?;
-
-  // TODO: make sure this is a local community
```

Comment:
Did you implement this check? I dont see it.

Reply:
Ah dang I didn't, but also I'm gonna redo a lot of these to consolidate them into the shared inbox.